test/suite/test_bug018.py

```diff
@@ -112,6 +112,11 @@ def subprocess_bug018(self):
                 self.conn = None
 
     def test_bug018(self):
+        # This test spawns another python instance but that circumvents the LD_PRELOAD logic in
+        # init.py, which means that python instance crashes. We could fix that but it would require
+        # some custom logic which is overkill for this single test.
+        if os.environ.get("TESTUTIL_TSAN") == "1":
+            self.skipTest("Not compatibable with TSan")
```

Comment:
Suggest moving this section of new code down below the func docstring on line 120.

Reply:
Fixed in [da7eeb8](https://github.com/wiredtiger/wiredtiger/pull/10839/commits/da7eeb890d255db94ed326c30f637f1302f31e91)